<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2f3136;</string>
  </property>
  <widget class="QTextBrowser" name="tb_chat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color: #ffffff;
color: #ffffff;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>11</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9764ff;
border: none;
color: white;
border-radius: 5px;
text-align: center;
text-decoration: none;
display: inline-block;
transition-duration: 0.4s;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_chat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
border-radius: 15px;
outline: none;
padding-left: 10px;
background-color: rgb(233, 233, 233);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Friend List</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>131</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color: #ffffff;
color: #ffffff;
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>129</width>
      <height>549</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_chatName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>대화상대 : all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
